notifications/src/main/java/dk/shape/games/notifications/presentation/SubjectNotificationsInteractor.kt

```diff
@@ -116,12 +122,14 @@ class SubjectNotificationsInteractor(
                 }
                 onSuccess()
             }
-        } catch (e: IOException) {
-            onFailure(e)
-            if (notificationsEventHandler is SubjectNotificationsEventHandler.Full.State) {
-                notificationsEventHandler.onNotificationsLoading(false)
-                notificationsEventHandler.onPreferencesSavedError(e)
-            }
+        } catch (e: Exception) {
```

Comment:
```suggestion
        } catch (e: IOException) {
            onFailure(e)
        } catch (e: HttpException) {
            onFailure(e)
        }
```
that way we don't need to throw the exception again